spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/SparkCatalog.java

```diff
@@ -83,7 +89,8 @@
 
   private String catalogName = null;
   private Catalog icebergCatalog = null;
-  private boolean cacheEnabled = true;
+  private boolean cacheEnabled = CatalogProperties.TABLE_CACHE_ENABLED_DEFAULT;
+  private long cacheExpirationIntervalMs = CatalogProperties.TABLE_CACHE_EXPIRATION_INTERVAL_MS_DEFAULT;
```

Comment:
Do these need to be fields? Can we remove them and just make them local within `initialize`?

Reply:
I use them in the Spark test, but I can assert on properties of the CachingCatalog underneath instead.

Reply:
Actually `cacheEnabled` is used in several other places. It needs to be a field. But I can remove `cacheExpirationIntervalMs`

Reply:
I removed `cacheExpirationIntervalMs` and made it a local variable inside `initialize`. The rest is left as it was before this PR.